cmake/TaichiCore.cmake

```diff
@@ -88,6 +88,9 @@ if (TI_WITH_CUDA)
     target_link_libraries(${LIBRARY_NAME} ${llvm_ptx_libs})
 endif()
 
+target_link_libraries(${LIBRARY_NAME} /usr/lib/libGLEW.so GLEW)
```

Comment:
Let's guard this part with `TI_WITH_OPENGL` so that people without OpenGL can build Taichi, just like `TI_WITH_CUDA` :-)